lib/facets/create-documents-facets.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function createDocumentFacets (results) {
+  var archiveValues = [];
+  results.aggregations.documents.archive.archive_filters.buckets.forEach(value => {
```

Comment:
yes the buckets property are defined by aggs-document and will be always defined. I change the forEach to a map!